go/tricorder/setup.go

```diff
@@ -20,18 +20,20 @@ func timeValToDuration(val *wrapper.Timeval) time.Duration {
 
 func initDefaultMetrics() {
 	programArgs := getProgramArgs()
-	var memStats runtime.MemStats
+	var totalMemory uint64
 	memStatsGroup := NewGroup()
 	memStatsGroup.RegisterUpdateFunc(func() time.Time {
+		var memStats runtime.MemStats
 		runtime.ReadMemStats(&memStats)
+		totalMemory = memStats.Sys - memStats.HeapReleased
```

Comment:
Consider checking that memStats.HeapReleased <= memStats.Sys here. It most likely will be, but in case it isn't, you want totalMemory to be 0 rather than some really large number.

Reply:
Done.